lib/meta_tags/text_normalizer.rb

```diff
@@ -40,6 +40,7 @@ def normalize_title(site_title, title, separator, reverse = false)
     def normalize_description(description)
       return '' if description.blank?
       description = cleanup_string(description)
+      return description if MetaTags.config.description_limit.to_i == 0
```

Comment:
Maybe move this condition to `truncate`? There is already `if limit` modifier, maybe change it to `if limit.to_i > 0` ?

Reply:
Hi, I would move this check then in a separate private method, for example - `description_limit_enabled?`. And did the same with the title limit.What do you think about this?

Reply:
Both description and title should follow the same rules: https://github.com/kpumuk/meta-tags/blob/master/lib/generators/meta_tags/templates/config/initializers/meta_tags.rb#L4

So I would fix it in `truncate` method as it is used for both.

Reply:
I think it is possible. But you need to write more tests

Reply:
Definitely, I will add it to my todo, thank you

Reply:
But you have to pass an additional argument to the method, for example `truncate_enable?`, I do not know how true this is.

Reply:
The idea was that limit would define whether we need to truncate, and how many characters to truncate to. So it checks if `limit` was supplied, but I forgot about `0` :-) So yes, checking if `limit` is `0` or `nil` is the right way